src/Model/ValueObject/Identity/UUID.php

```diff
@@ -6,25 +6,33 @@
 
 use CultuurNet\UDB3\Model\ValueObject\String\Behaviour\IsString;
 use CultuurNet\UDB3\Model\ValueObject\String\Behaviour\MatchesRegexPattern;
+use Ramsey\Uuid\Uuid as RamseyUuid;
 
 class UUID
 {
     use IsString;
     use MatchesRegexPattern;
 
+    public const NIL = '00000000-0000-0000-0000-000000000000';
+
     /**
      * Ensures backwards compatibility with older, malformed, uuids present in UDB.
      */
-    public const BC_REGEX = '\\A[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-?[0-9A-Fa-f]{12}\\z';
+    public const BC_REGEX = '/\\A[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-?[0-9A-Fa-f]{12}\\z/';
```

Comment:
Maybe we need to make this private because it is nowhere used?

Reply:
I use it in the new test tests/Model/ValueObject/Identity/UUIDTest.php.

Reply:
I don't think that is needed. You could just create a valid and invalid UUID?